pallets/salp/src/lib.rs

```diff
@@ -196,28 +288,44 @@ pub mod pallet {
 		_,
 		Blake2_128Concat,
 		ParaId,
-		Option<FundInfo<T::AccountId, BalanceOf<T>, T::BlockNumber, LeasePeriodOf<T>>>,
-		ValueQuery
+		Option<FundInfo<AccountIdOf<T>, BalanceOf<T>, LeasePeriod>>,
```

Comment:
@AllenPocketGamer @ark930 seems LeasePeriod is not BlockNumber in crowdloan

Reply:
LeasePeriod is defined as BlockNumber in pallet auctions